src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @OneToOne
```

Comment:
```suggestion
    @ManyToOne
    @JoinColumn(name = "country_id")
```
Try to understand why this is not one to one relationship. Can the country contain only one actor?